single.php

```diff
@@ -1,37 +0,0 @@
-<?php
```

Comment:
I just realized this change is out-of-scope for the pull request. There is a separate issue to deprecate post formats, which may allow us to remove this file. However, I suspect we will still want to define a `single.php` in the `creativecommons-base` project, so long as the `wp-theme-creativecommons.org` theme `single.php` takes precedence when enabled.

Reply:
Okay this means I will have to port the footer elements over to the `single.php` file.  I think previously the `single.php` file was made up of different templates (author, header, footer)...I just maintained the templates. If they are to be deprecated then I will have to move them over to the `single.php` file.

I am referring to this folder: https://github.com/creativecommons/creativecommons-base/tree/main/inc/partials/entry

Reply:
It is alright for the `creativecommons-base` and `wp-theme-creativecommons.org` to have different blog post layouts. The Figma mockups only pertain to the `wp-theme-creativecommons.org` in cases like this.

Reply:
> I am referring to this folder: https://github.com/creativecommons/creativecommons-base/tree/main/inc/partials/entry

I have proposed that we deprecate these post formats https://github.com/creativecommons/project_creativecommons.org/issues/165